universal-application-tool-0.0.1/test/models/QuestionTest.java

```diff
@@ -0,0 +1,61 @@
+package models;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.google.common.collect.ImmutableMap;
+import java.util.Locale;
+import java.util.Optional;
+import org.junit.Test;
+import repository.QuestionRepository;
+import repository.WithResettingPostgresContainer;
+import services.question.QuestionDefinition;
+
+public class QuestionTest extends WithResettingPostgresContainer {
+
+  @Test
+  public void canSaveQuestion() {
+    QuestionRepository repo = app.injector().instanceOf(QuestionRepository.class);
+    QuestionDefinition definition =
+        new QuestionDefinition(1L, 1L, "test", "my.path", "", ImmutableMap.of(), Optional.empty());
+    Question question = new Question(definition);
+
+    question.save();
+
+    Question found = repo.lookupQuestion(definition.getId()).toCompletableFuture().join().get();
+
+    assertThat(found.getQuestionDefinition().getId()).isEqualTo(definition.getId());
```

Comment:
nit(non-blocking): I like to see the `.isEqualTo` values as raw values when possible.

Reply:
ditto for the `canSerializeLocalizationMaps` test case.

Reply:
Sounds good to me! Will update tests